packages/components/collapse/src/collapse.vue

```diff
@@ -57,6 +57,9 @@ export default defineComponent({
       () => props.modelValue,
       () => {
         activeNames.value = [].concat(props.modelValue)
+      },
+      {
+        deep: true,
       }
```

Comment:
Do you think it is possible to watch like this:

```ts
watch(() => props.modelValue.length, handler)
```

instead of adding `deep`?

Reply:
Performance of `deep` is indeed not as good as `watch`.

When `length` is not changed but the item changed, it can't be monitoring, so should we watch both `props.modelVaule` and `props.modelValue.length`?

Reply:
Let's use deep here for now, if there is any perf concern we can try to fix it later.